internal/collector/processor/processor.go

```diff
@@ -34,29 +31,6 @@ type SpanProcessor interface {
 	// TODO: (@pjanotti) For shutdown improvement, the interface needs a method to attempt that.
 }
 
-// An initial processor that does not sends the data to any destination but helps debugging.
-type debugSpanProcessor struct{ logger *zap.Logger }
-
-var _ SpanProcessor = (*debugSpanProcessor)(nil)
-
-func (sp *debugSpanProcessor) ProcessSpans(td data.TraceData, spanFormat string) error {
-	if td.Node == nil {
-		sp.logger.Warn("Received batch with nil Node", zap.String("format", spanFormat))
```

Comment:
I kind of miss that enabling the debug exporter we were able to log the format of the spans reaching the service, I guess we will go with metrics to identify that now...

Reply:
+1 metrics will be good.